.github/workflows/cd.yml

```diff
@@ -17,20 +17,10 @@ jobs:
     strategy:
       matrix:
         include:
-          # Supported `cross` targets:
-          #   https://github.com/rust-embedded/cross#supported-targets
-
-          # Linux targets
-          - { target: x86_64-unknown-linux-gnu, os: ubuntu-latest, cross: false }
-          - { target: x86_64-unknown-linux-musl, os: ubuntu-latest, cross: true }
-          - { target: aarch64-unknown-linux-musl, os: ubuntu-latest, cross: true }
```

Comment:
Is dropping support for `aarch64-unknown-linux` necessary? I.e. can it not be built with glibc?

I think it's fine (also see https://github.com/hannobraun/Fornjot/issues/369#issuecomment-1070784881), just asking for clarification.

Reply:
I missed to check that; there should be a GNU target for ARM on Linux though.

Reply:
I think it's probably worth supporting, if it's easy. If not, I don't think it's worth spending the effort right now. I've opened #371 to track that.

Thanks!